e2e/cypress/integration/02_global_filter.spec.js

```diff
@@ -0,0 +1,76 @@
+describe('global projects filter', () => {
+  const nonAdminUsername = "cypressTestUserNonAdmin"
```

Comment:
Test failed on my box until I fixed this:
```suggestion
  const nonAdminUsername = "cypresstestusernonadmin"
```
Local storage apparently converted it all to lower case.

Reply:
yes, I thought i fixed that, too . . . I must have not pushed something . . .

Reply:
yep 😳